src/Core/Logging/Extensions/ILogServiceExtensions.Internal.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceExtensions
+    {
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(DateTimeOffset, LogLevel, TState, string)"./>
+        /// </summary>
+        private static void Internal<TState>(
```

Comment:
Codacy found an issue: [Method has 6 parameters, which is greater than the 4 authorized.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)